src/pybind/mgr/mgr_module.py

```diff
@@ -733,10 +740,24 @@ def get_all_perf_counters(self, prio_limit=PRIO_USEFUL):
                     if counter_schema['priority'] < prio_limit:
                         continue
 
-                    counter_info = counter_schema
-                    counter_info['value'] = self.get_latest(service['type'], service['id'],
-                                                            counter_path)
-                    result[svc_full_name][counter_path] = counter_info
+                    # Also populate count for the long running avgs
+                    if counter_schema['type'] & self.PERFCOUNTER_LONGRUNAVG:
+                        counter_info = dict(counter_schema)
```

Comment:
could extract `counter_info = dict(counter_schema)` and `result[svc_full_name][counter_path] = counter_info` out of this `if-else` block.